docs/tutorials/data_visualization.ipynb

```diff
@@ -241,36 +349,98 @@
+     },
+     "execution_count": 21,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
     "titanic_numerical.plot_histograms()"
    ],
+   "metadata": {
+    "collapsed": false,
+    "ExecuteTime": {
+     "start_time": "2023-06-16T08:16:56.298442Z",
+     "end_time": "2023-06-16T08:16:58.271293Z"
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Let's look at the different histograms of certain columns:\n",
+    "* According to the `age` histogram, a majority of the passengers were aged between 20 and 40. If the bars within this range are added up, about 750 passengers fall between this age range. Just a few passengers were above 75 years.\n",
+    "    * It is worth noting, that we can use the `normal distribution` here to approximate the data and future measurements. The normal distribution is a symmetric bell-shaped curve with a continuous range from -infinity to +infinity and has a common value for mean, median, and mode.\n",
```

Comment:
* nothing is spelled wrong

Reply:
It's correct. "worth noting" means "it is important to note".